core/keyserver/keyserver.go

```diff
@@ -102,8 +103,7 @@ func (s *Server) getEntry(ctx context.Context, userID string, epoch int64) (*tpb
 
 	neighbors, err := s.tree.NeighborsAt(ctx, index[:], epoch)
 	if err != nil {
-		log.Printf("Cannot get neighbors list: %v", err)
```

Comment:
Hmm... Let's chat more about this Monday. I worry that the exported / unexported rule will result in a lot of error conversion code.  I think I prefer the server module / non-server-module deliniation